timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -226,35 +216,56 @@ def get_preset_models(
             logger.log(20, f"Custom Model Type Detected: {model}")
             model_type = model
 
-        model_type_kwargs = dict(
-            path=path,
-            freq=freq,
-            prediction_length=prediction_length,
-            eval_metric=eval_metric,
-            eval_metric_seasonal_period=eval_metric_seasonal_period,
-            hyperparameters=model_hps,
-            **kwargs,
-        )
+        model_hps_list = hyperparameters[model]
+        if not isinstance(model_hps_list, list):
+            model_hps_list = [model_hps_list]
+
+        for model_hps in model_hps_list:
+            ag_args = model_hps.pop(ag.constants.AG_ARGS, {})
+            for key in ag_args:
+                if key not in VALID_AG_ARGS_KEYS:
+                    logger.warning(f"WARNING: Unknown ag_args key: {key}")
+            model_name_base = get_model_name(ag_args, model_type)
+
+            model_type_kwargs = dict(
+                name=model_name_base,
+                path=path,
+                freq=freq,
+                prediction_length=prediction_length,
+                eval_metric=eval_metric,
+                eval_metric_seasonal_period=eval_metric_seasonal_period,
+                hyperparameters=model_hps,
+                **kwargs,
+            )
 
-        # add models while preventing name collisions
-        model = model_type(**model_type_kwargs)
-        name_stem = model.name
+            # add models while preventing name collisions
+            model = model_type(**model_type_kwargs)
 
-        model_type_kwargs.pop("name", None)
-        increment = 1
-        while model.name in all_assigned_names:
-            increment += 1
-            model = model_type(name=f"{name_stem}_{increment}", **model_type_kwargs)
+            model_type_kwargs.pop("name", None)
+            increment = 1
+            while model.name in all_assigned_names:
+                increment += 1
+                model = model_type(name=f"{model_name_base}_{increment}", **model_type_kwargs)
```

Comment:
See tabs from the review, can you apply black formatter?

Reply:
I think this is an intentional change - there a new for loop that moved the indentation one level deeper. 

`black` is already a part of the `lint_check` CI step, so this code should be formatted correctly.